radio/apps/programmes/urls.py

```diff
@@ -31,11 +31,12 @@
        ),
        name='list'
    ),
+   url(r'^json', views.programmes_json, name='programmes_json'),
```

Comment:
I have doubts if this function should be moved to the api/1/

You can check how functions are defined in Lines 72-73 at /dadio/configs/commons/urls.py
https://github.com/iago1460/django-radio/blob/638e7be4d160b9d4cd7691b430226dd0e9a92341/radio/configs/common/urls.py#L72-L73

I would like @iago1460 recommends where this function should be available.